sentry/src/main/java/io/sentry/cache/EnvelopeCache.java

```diff
@@ -58,6 +58,8 @@ public final class EnvelopeCache extends CacheStrategy implements IEnvelopeCache
   public static final String CRASH_MARKER_FILE = "last_crash";
   public static final String NATIVE_CRASH_MARKER_FILE = ".sentry-native/" + CRASH_MARKER_FILE;
 
+  public static final String STARTUP_CRASH_MARKER_FILE = "startup_crash";
```

Comment:
Could we run into a case where people want to disable the cache but still keep the startup crash feature on?

Reply:
I guess we just say we don't support this case, because we need the envelope to be stored in cache in the first place.. supporting it would mean we either block on unhandled exception or send the envelopes in a different process, which is a whole different story